joomla/joomla-cms
# [4.0] Fix switcher position.

### Testing Instructions
1. Navigate to "System - Site Templates - Cassiopeia Details and Files".
2. Create an override for a component view.
3. Edit the override.
4. Look at the switcher in disabled and enabled state.
5. Apply the PR.
6. Make sure that the switcher still looks correctly in other places, like com_config.


### Expected result
![Screenshot_2019-09-19 Templates Customise (Cassiopeia) - Joomla 4 Git - Administration](https://user-images.githubusercontent.com/1410135/65255485-45da6600-dafe-11e9-97b0-dcdf8025419a.png)



### Actual result
![Screenshot_2019-09-19 Templates Customise (Cassiopeia) - Joomla 4 Git - Administration broken](https://user-images.githubusercontent.com/1410135/65255545-61de0780-dafe-11e9-9ab5-44828c01688e.png)
